collections.md

```diff
@@ -124,3 +124,56 @@ The json representation is as follow
 ```
 see also the [ResourcePolicies endpoint](resourcepolicies.md)
 
+## Creating a collection
+
+**POST /api/core/collections?parentCommunity=<:communityUUID>**
```

Comment:
Can I suggest to use parent as parameter name instead than parentCommunity? it will be nice to define an uniform naming strategy for our parameter (and include such information below https://github.com/DSpace/Rest7Contract#on-the-naming-of-endpoints )

parent is shorter and communities and collections could eventually collapse in a single entity in a future version

Reply:
The parameter has been renamed. There's no potential ambiguity in the name parent since no different type of parent can be specified